src/routes/hud/elements/notifications/Notification.svelte

```diff
@@ -21,21 +21,21 @@
             "a c";
     grid-template-columns: max-content 1fr;
     column-gap: 10px;
-    background: rgba(0, 0, 0, 0.4);
-    border-radius: 60px;
+    background: rgba(0, 0, 20, 0.4);
```

Comment:
It would be better to keep it at black so that it fits with all colours.

Reply:
did I changed that xdd don't notice